### ISSUE TITLE
Searching Keyboard Updates

### ISSUE THREAD

**Issue by delianides** [NewSpring/Apollos#679]
The keyboard remains active when someone is searching on the discover page.

- [ ] Discover
- [ ] Group Finder
- [x] Determine if "Enter" can be added to native keyboard
- [ ] add if we want

**Comment by johnthepink** [comment]
as @benwiley pointed out, this should be as easy as disabling focus on the text field after submitting

**Comment by johnthepink** [comment]
We should include Group Finder search in this as well.

**Comment by johnthepink** [comment]
iOS does give you access to setting a custom string for the "return" button at the native level, or in a web view. We can use predefined values for that field.

At the web view level, this involves:

1. wrapping text inputs in a <form> tag
2. adding an action attribute to the <form>
3. ensuring the action doesn't do anything on form submission
4. setting an appropriate text input type on form elements

Here are the possibilities at the web view level:

## Go

![screenshot 2016-07-31 16 50 10](https://cloud.githubusercontent.com/assets/816517/17279415/6671a39a-5741-11e6-8b26-69076aea7ee7.png)

## Search

![screenshot 2016-07-31 16 50 27](https://cloud.githubusercontent.com/assets/816517/17279418/703dc93a-5741-11e6-8d84-cfcbf7ceeae7.png)

## Return

![screenshot 2016-07-31 16 52 18](https://cloud.githubusercontent.com/assets/816517/17279420/7641f086-5741-11e6-86a6-5606bfe331a2.png)

I'm not sure what these do for third party keyboards. Probably assumes they will implement the same way.

At the native level, you can use the native `UITextInputTraits` on a `UITextField` or `UITextView` element. Documentation [here](https://developer.apple.com/library/ios/documentation/UIKit/Reference/UITextInputTraits_Protocol/#//apple_ref/c/tdef/UIReturnKeyType). 

Basically, this amounts to "return", "Go", "Google", "Join", "Next", "Route", "Search", "Send", "Yahoo", "Done", "Emergency Call", "Continue". Using one of these that isn't available in the web view will involve rendering a native text input over the top of the web view, and communicating back and forth. Messy, not scalable, and IMO not worth it.

**Comment by jbaxleyiii** [comment]
@johnthepink this is great! Lets go with search
